llvm/include/llvm/CGData/StableFunctionMap.h

```diff
@@ -42,8 +42,8 @@ struct StableFunction {
   StableFunction(stable_hash Hash, const std::string FunctionName,
                  const std::string ModuleName, unsigned InstCount,
```

Comment:
Shouldn't the `const std::string` be `StringRef`?

Reply:
I didn't make that change but if it is required, I can fix it here or in another PR.

Reply:
Either is fine

Reply:
I've opened a new PR https://github.com/llvm/llvm-project/pull/118749 for the change, I'll remove the move from here as they are not required now.